go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/fatih/structs v1.1.0 // indirect
 	github.com/google/uuid v1.1.1
 	github.com/mattermost/mattermost-plugin-autolink v1.1.3-0.20200203183014-8c82b7dc7fa6
-	github.com/mattermost/mattermost-plugin-workflow-client v0.0.0-20200121183617-b71061053ec5
+	github.com/mattermost/mattermost-plugin-workflow-client v0.0.0-20200225165807-b0524a245156
```

Comment:
Should we start versioning this, if only to make it easier to upgrade in go.mod?

Reply:
Ya. Want to move this stuff back to the main repository so not bothering until then.